R/model.R

```diff
@@ -129,8 +129,8 @@
 #' fit_optim_w_init_list$init()
 #' }
 #'
-cmdstan_model <- function(stan_file, compile = TRUE, ...) {
-  CmdStanModel$new(stan_file = stan_file, compile = compile, ...)
+cmdstan_model <- function(stan_file, dir = NULL, compile = TRUE, ...) {
```

Comment:
This is fine, but for consistency I wonder if `dir` should be an argument to the `compile()` method that can get passed through `cmdstan_model()` via `...` like the other arguments to `compile()`. What do you think?

Reply:
Agreed